lib/audited/auditor.rb

```diff
@@ -324,6 +330,13 @@ def audit_update
         end
       end
 
+      def audit_touch
+        unless (changes = audited_changes(for_touch: true)).empty?
+          write_audit(action: "update", audited_changes: changes,
```

Comment:
The `action` could be set to `"touch"`, but I left it as an `"update"` since the underlying change is effectively updating a timestamp from 1 value to the current time.

Reply:
I think you made the right call.